js_modules/dagster-ui/packages/ui-core/src/overview/OverviewTimelineRoot.tsx

```diff
@@ -49,8 +49,13 @@ export const OverviewTimelineRoot = ({Header, TabButton}: Props) => {
     defaults: {search: ''},
   });
 
+  const isInitialRender = React.useRef(true);
   React.useEffect(() => {
-    setNow(Date.now());
+    if (isInitialRender.current) {
+      isInitialRender.current = false;
+    } else {
+      setNow(Date.now());
+    }
```

Comment:
can you explain this fix? we don't want to set now on the initial render but we do the subsequent ones?

Reply:
This intended purpose of this `useEffect` is to update `now` whenever `hourWindow` changes in order to trigger a refetch.

Because of the way `useEffect` works it also fires on initial render (since hourWindow was previously undefined) and causes a double fetch only on initial render.